virtual_rainforest/models/animals/constants.py

```diff
@@ -140,8 +152,13 @@
 
 TEMPERATURE: float = 37.0  # Toy temperature for setting up metabolism [C].
 
-REPRODUCTION_ENERGY_MULTIPLIER: float = 1.5  # Toy value for thresholding reproduction
-REPRODUCTION_ENERGY_COST_MULTIPLIER: float = 0.5  # Toy value for reproduction costs
+BIRTH_MASS_THRESHOLD: float = 1.5  # Toy value for thresholding reproduction.
+
+FLOW_TO_REPRODUCTIVE_MASS_THRESHOLD: float = (
+    1.0  # Toy value for threshold of trophic flow to reproductive mass.
+)
+
+DISPERSAL_MASS_THRESHOLD: float = 0.75  # Toy value for thesholding dispersal.
```

Comment:
In general it's better to describe constants with docstrings cf. comments. That way sphinx will be able to pick them up (as will your IDE).

```suggestion
DISPERSAL_MASS_THRESHOLD: float = 0.75
"""Toy value for thesholding dispersal."""
```

Reply:
(repeating myself just so it's clear what I'm doing)
 I'll be doing a full constants rework after this PR to bring everything in line, including docstrings.